src/main/resources/avro/genotypephenotypemethods.avdl

```diff
@@ -0,0 +1,115 @@
+
+@namespace("org.ga4gh")
+/**
+This protocol defines genotype to phenotype relationships
+query system.
+
+The goal here is to allow users to query systems using one
+or more of Genotype, Phenotype, Environment, and Evidence.
+
+A query using one of the above items is to mean, by default,
+that the remainder of the query is as a "wildcard", such
+that all matches to just that query term would come back.
+Combinations of the above are to act like AND rather than OR.
```

Comment:
The current code permits only one evidence in the Association, and the Query also only allows for one Query (as opposed to an array) which would be ANDed to the Phenotype+Feature...so at the moment a user could not query with multiple Evidences.  But this suggests that we should update the code to 1) make the Evidence part of an association as an array of Evidences, and 2) that the user can then also query with an array of multiple evidences, if desired, to be consistent with the other Query types.